### ISSUE TITLE
Anything I can do to maintain this repo?

### ISSUE THREAD

**Issue by ninn-kou** [zziz/pwc#42]
This repo is fantastic but it seems the owner suspended maintenance. Any updating fork there? Or if there is anything I can do to help with maintenance?

**Comment by pquodling** [comment]
There is an old adage that one of my mentors used to say - "he who proposes, disposes" - i.e. if you think it should be done, don't just say so, do it. The original owner, like many of us, has a busy life (In this case, as an ML Devo at amazon), although it appears that this repo was started well before that in his career, perhaps when he a) had more time, and b) was more focused on the academic side of research that included code. 

Github is open to input from anyone - This is a repo of "pointers to code", so it doesn't even need validated or tested code, or skills relating to same from the contributor (although that has some benefits). - I would suggest adding edits to the repo for items that you see that add to the body of information in the Repo - or, as the readme suggests, tweet the owner, to offer to take over.  It appears that in spite of a "daily addition target", it has languished for a while, so perhaps starting back in the time window when it started to "go quiet" would be a start. 

Equally, perhaps the owner can add some of the methodologies used to search for candidates for this, so others can take over operating them. (Never reinvent a perfectly good wheel).

**Comment by ninn-kou** [comment]
> There is an old adage that one of my mentors used to say - "he who proposes, disposes" - i.e. if you think it should be done, don't just say so, do it. The original owner, like many of us, has a busy life (In this case, as an ML Devo at amazon), although it appears that this repo was started well before that in his career, perhaps when he a) had more time, and b) was more focused on the academic side of research that included code.
> 
> Github is open to input from anyone - This is a repo of "pointers to code", so it doesn't even need validated or tested code, or skills relating to same from the contributor (although that has some benefits). - I would suggest adding edits to the repo for items that you see that add to the body of information in the Repo - or, as the readme suggests, tweet the owner, to offer to take over. It appears that in spite of a "daily addition target", it has languished for a while, so perhaps starting back in the time window when it started to "go quiet" would be a start.
> 
> Equally, perhaps the owner can add some of the methodologies used to search for candidates for this, so others can take over operating them. (Never reinvent a perfectly good wheel).

Yes, thank you for your suggestion 🥰! You are right, do something physically is greater. However, it seems that after 2018, the contributors did not approve pull requests anymore. Just wondering if there is any way to be a maintainer.

**Comment by gusferreira1203** [comment]
Since when am I a contributor? kkkkk
